views/activities-new.jade

```diff
@@ -2,15 +2,16 @@ extends layout
 
 block content
     .row
-        .col-sm-6.offset-sm-6
             form(action='/itineraries/#{itineraryId}/activities/new')
+              center
```

Comment:
The center tag is not supported in HTML5. https://www.w3schools.com/tags/tag_center.asp
It appears that you have a myclaas class that centers things.